### PR TITLE
[stable/pachyderm] Added ability to deploy the pachyderm dashboard

### PR DESCRIPTION
**Pull request by LaurentGoderre**
Signed-off-by: Laurent Goderre <laurent.goderre@gmail.com>

#### What this PR does / why we need it:
Allows deploying the pachyderm dashboard

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/pachyderm/templates/dash_deployment.yaml
Comment (Aisuko) [inline stable/pachyderm/templates/dash_deployment.yaml:11]: Should we add `annotations` here?

File: stable/pachyderm/Chart.yaml
Comment (Aisuko) [inline stable/pachyderm/Chart.yaml:13]: `0.3.0` could make sense.

File: stable/pachyderm/templates/dash_deployment.yaml
Comment (Aisuko) [inline stable/pachyderm/templates/dash_deployment.yaml]: Should we add the blank row at the end, that could help to across the CI test.

File: stable/pachyderm/templates/dash_ingress.yaml
Comment (Aisuko) [inline stable/pachyderm/templates/dash_ingress.yaml]: Same as above.

File: stable/pachyderm/templates/dash_svc.yaml
Comment (Aisuko) [inline stable/pachyderm/templates/dash_svc.yaml]: Same as above.

File: stable/pachyderm/templates/pachd_svc.yaml
Comment (Aisuko) [inline stable/pachyderm/templates/pachd_svc.yaml:8]: `{{ toYaml .Values.pachd.service.annotations | indent 4 }}` could be make sense right?

File: stable/pachyderm/values.yaml
Comment (Aisuko) [inline stable/pachyderm/values.yaml]: `annotations: {}` could be make sense right?

**Review by Aisuko — CHANGES_REQUESTED** [review]
Thanks for your PR, there may some piece of advice. Oh, please signoff DCO

**Comment by jonandernovella** [comment]
Looks good to me. Those minor changes should do it @LaurentGoderre

File: stable/pachyderm/templates/dash_deployment.yaml
Comment (LaurentGoderre) [reply stable/pachyderm/templates/dash_deployment.yaml:11]: Neither the `pachd` or `etcd` deployments have annotations there so I wouldn't add it in this PR.

**Review by jonandernovella — APPROVED** [review]
LGTM!

**Comment by LaurentGoderre** [comment]
It isn't quire ready, it deploys but the UI doesn't work because of the web sockets.

**Review by jonandernovella — CHANGES_REQUESTED** [review]
We first need to fix the web sockets issue before merging this PR.

**Comment by zanhsieh** [comment]
up

**Comment by jonandernovella** [comment]
@LaurentGoderre Your second commit needs signoff before we can marge it.

**Pull request closed without merging**
